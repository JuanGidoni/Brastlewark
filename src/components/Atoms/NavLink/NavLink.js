import { Link } from 'react-router-dom';

const NavLink = ({ text, to, className }) => {
 return (
  <Link to={to} className={`nav-link ${className ? className : ''}`}>
   {text}
  </Link>
 )
}

export default NavLink;
