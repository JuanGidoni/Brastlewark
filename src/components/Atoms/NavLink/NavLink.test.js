import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NavLink from './NavLink';

it('renders correctly', () => {
 const tree = renderer
  .create(<Router><NavLink to="/" text="Example" className="example" /></Router>)
  .toJSON();
 expect(tree).toMatchSnapshot();
});