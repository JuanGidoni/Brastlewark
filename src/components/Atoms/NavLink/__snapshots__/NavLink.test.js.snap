// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="nav-link example"
  href="/"
  onClick={[Function]}
>
  Example
</a>
`;
